README.md

```diff
@@ -188,7 +187,20 @@ Wrangler can be installed both through [npm](https://www.npmjs.com/get-npm) and
     ```
     $ brew install openssl@1.1
     ```
-    
+
+1. Updating `wrangler:
+
+    To get the latest version of Wrangler, using Cargo, run:
+
+    ```bash
+    cargo install --git https://github.com/cloudflare/wrangler --force
+    ```
+
+   To get the latest version of Wrangler, using NPM, run:
+
+   ```bash
+   npm install @cloudflare/wrangler
```

Comment:
```suggestion
   npm install -g @cloudflare/wrangler
```

do we install it globally? installing it locally will expose the executable only locally to npm at least.